istio/istio.go

```diff
@@ -424,6 +446,37 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 		return &meshes.ApplyRuleResponse{
 			OperationId: arReq.OperationId,
 		}, nil
+	case installHipsterShopCommand:
+        go func() {
+            opName1 := "deploying"
+            if arReq.DeleteOp {
+                opName1 = "removing"
+            }
+            if err := iClient.executeHipsterShopInstall(ctx, arReq); err != nil {
```

Comment:
this should invoking your method above (OR) your function in `getting_microservices-release.go`